app/src/main/java/com/quran/labs/androidquran/data/QuranInfo.java

```diff
@@ -126,6 +128,16 @@ public int getSuraNumberFromPage(int page) {
     return sura;
   }
 
+  public List<Integer> getListOfSurahWithStartingOnPage(int page) {
+    List<Integer> result = new ArrayList<>();
```

Comment:
we can make this more efficient if we first do something like:

```int startSura = pageSuraStart[page - 1];```

then start the loop below from `i = startSura` instead of 0. also, once you hit a sura with the start page > page, break out of the loop.